### PR TITLE
Rename lists in the OPA example

### PR DESCRIPTION
**Pull request by stanlyDoge**
Signed-off-by: Stanislav Knot <sknot@redhat.com>

### Type of change
- Refactoring

### Description
Rename lists in the OPA example.

### Checklist

- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by stanlyDoge** [comment]
@strimzi-ci run tests profile=regression testcase=io.strimzi.systemtest.security.OpaIntegrationST

**Comment by stanlyDoge** [comment]
Did not found any.

File: systemtest/src/test/resources/opa/opa.yaml [hunk 1]
```diff
@@ -96,35 +96,35 @@ data:
     deny {
       is_topic_resource
       is_read_operation
-      not consumer_is_whitelisted
+      not consumer_is_allowlisted
```
Comment (scholzj) [inline systemtest/src/test/resources/opa/opa.yaml:99]: `consumer_is_allowlisted` and `producer_is_allowlisted` seem very weird to me. Maybe you can use `*_is_allowed` or `*_is_on_allowlist`.

**Comment by stanlyDoge** [comment]
@strimzi-ci run tests profile=regression testcase=io.strimzi.systemtest.security.OpaIntegrationST

**Pull request merged**